.travis.yml

```diff
@@ -62,24 +60,24 @@ before_script:
 
 script:
   - >
-    if [[ $TRAVIS_PYTHON_VERSION_MINOR < 4 || $TRAVIS_PYTHON_VERSION_MAJOR == 2 ]]; then
+    if [[ "$UNSUPPORTED" == true ]]; then
       coverage run setup.py install | grep -q 'coala supports only python 3.4 or later'
+    fi
+  - if [[ "$UNSUPPORTED" == true ]]; then exit; fi
```

Comment:
why?  Travis shows only the first line, so ideally the `exit` is there so the reader can see what is being done.
Why hide the `if`?